.travis.yml

```diff
@@ -0,0 +1,35 @@
+language: objective-c
+osx_image: xcode7.2
+branches:
+  only: master
+script: placeholder # workaround for https://github.com/travis-ci/travis-ci/issues/4681
+matrix:
+  include:
+    - script: ./build.sh verify-swiftlint
+      env: JOB=verify-swiftlint
+      before_install: brew install swiftlint
+    - script: ./build.sh verify-docs
+      env: JOB=verify-docs
+      before_install: gem install jazzy
```

Comment:
>I don't know if using the bundler cache would negatively affect other builds, so I'd rather not.

Beside CocoaPods, I don't see what else could be affected by that change.

>Also, the docs you linked to show to specify `language: ruby`, so I'm not sure how supported it would be to use it with `language: objective-c`.

Directly above that subsection, in the introduction of [the section](https://docs.travis-ci.com/user/caching/#Bundler), they explain that this is available for both type of images.